app/controllers/users_controller.rb

```diff
@@ -54,9 +51,8 @@ def edit
   end
 
   def update
-    params[:user][:email].downcase!
     @user.accessible = :all if current_user.admin?
```

Comment:
does strong_params make it so you don't need this anymore?  not sure what this is for